Recruit-Training/Basic-Recruit-Training/pointers/main.cpp

```diff
@@ -19,29 +20,35 @@ int main()
     int *p = &x;
 
     //TODO in the line below predict what what is going to be output
-    cout << "Predicted value of p: " /* << PLACE YOUR PREDICTION HERE*/ << endl;
+    cout << "Predicted value of p: 0xsomething" << endl;
```

Comment:
As mentioned in the instructions, please place your predicted values between `<< <<`.